source/Buoy-Tests.package/AssertionCheckerTest.class/instance/testEnforceOneFactFailingRaisingError.st

```diff
@@ -0,0 +1,8 @@
+tests
+testEnforceOneFactFailingRaisingError
+
+	| explanation |
+
+	explanation := 'A false statement'.
+	"For single fact checking any exception can be configured to be raised"
+	self should: [ AssertionChecker enforce: [ false ] because: explanation raising: Error ] raise: Error withExceptionDo: [ :exception | self assert: exception messageText equals: explanation ]
```

Comment:
The difference between this test and testEnforceOneFactFailingRaisingNonDefaultException is not clear.